ui/raidboss/data/04-sb/raid/o12s.js

```diff
@@ -39,14 +39,7 @@
       condition: function(data, matches) {
         return data.me == matches.target;
       },
-      alarmText: {
-        en: 'Don\'t Stack!',
-        de: 'Nicht stacken!',
-        fr: 'Ne vous packez pas !',
-        ja: 'スタックするな！',
-        cn: '分散站位！',
-        ko: '쉐어 맞지 말것',
-      },
+      response: Responses.spread(),
```

Comment:
I think this is confusing.  Everybody stacks but one person.  I think calling this "spread" is not as good as "don't stack".  Maybe this is better to just leave as-is.